xml/http_wwwauth.xml

```diff
@@ -1,18 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <fingerprints matches="http_header.wwwauth" protocol="http" database_type="service" preference="0.85">
   <!-- HTTP WWW-Authenticate headers are matched against these patterns to fingerprint HTTP servers. -->
-  <fingerprint pattern="^(?:Basic|Digest) realm=.[iI]RMC(?:@(IRMC[0-9a-fA-F]{6}))?..*$">
-    <description>Fujitsu Siemens Primergy with BMC RemoteView on an iRMC card</description>
-    <param pos="0" name="service.vendor" value="Fujitsu Siemens"/>
-    <param pos="0" name="service.product" value="RemoteView"/>
-    <param pos="0" name="service.family" value="RemoteView"/>
-    <param pos="0" name="hw.vendor" value="Fujitsu Siemens"/>
-    <param pos="0" name="hw.family" value="Primergy"/>
-    <param pos="0" name="hw.product" value="Primergy"/>
-    <param pos="1" name="host.name"/>
-  </fingerprint>
-  <fingerprint pattern="^(?:Basic|Digest) realm=.access.$">
+  <fingerprint pattern="^(?:Basic|Digest) realm=&quot;access&quot;$">
```

Comment:
Hmm, will it always be double quotes?  Perhaps that it is why historically `.` has been used to may be match single or double quotes?

Reply:
I should have worded that differently. I had replaced them opportunistically.  I've gone back now and replaced them all. Thanks for the feedback.

Reply:
Good point. We actually have examples of this in our data set but none that are currently covered by the existing database.

Reply:
And I immediately see that I'm wrong.

Reply:
Should be sorted out now